tests/jenkins/envutils.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
Is this required? I would prefer to use minimal setup here!

Reply:
You are right. It is not needed and will be removed :+1: